src/script_chainer/gui/page/script_setting_interface.py

```
@@ -378,30 +461,175 @@ def on_edit_clicked(self) -> None:
+        try:
+            launch_in_terminal(
+                command=[python, path],
+                cwd=os.path.dirname(path) or None,
+                title=f'运行 {os.path.basename(path)}',
+            )
+            _show_success(self.window(), '运行', f'已启动 {os.path.basename(path)}')
+        except Exception as e:
+            _show_error(self.window(), '运行失败', str(e))
+
+    def on_edit_clicked(self) -> None:
+        """编辑 Python 脚本。外部脚本直接调用外部编辑器，内部脚本弹窗编辑。"""
```

Ruff (0.15.10)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)